content/influxdb3/core/reference/config-options.md

```diff
@@ -178,6 +179,25 @@ configuration--for example, the same bucket.
 
 ---
 
+#### query-file-limit
+
+Limits the number of Parquet files a query can access. Default is `432` (72 hours).
```

Comment:
```suggestion
Limits the number of Parquet files a query can access. Default is `432`. With the default setting of gen1 time blocks of 10 minutes, this can be up to 72 hours, but could be less depending on gen1 configuration and whether all data for a given 10 minute block of time was ingested during the same period.
```